tests/ui/target_modifiers/defaults_check.error.stderr

```diff
@@ -5,8 +5,8 @@ LL | #![feature(no_core)]
    | ^
    |
    = help: the `-Zreg-struct-return` flag modifies the ABI so Rust crates compiled with different values of this flag cannot be used together safely
-   = note: `-Zreg-struct-return=true` in this crate is incompatible with `-Zreg-struct-return=` in dependency `default_reg_struct_return`
-   = help: set `-Zreg-struct-return=` in this crate or `-Zreg-struct-return=true` in `default_reg_struct_return`
+   = note: `-Zreg-struct-return=true` in this crate is incompatible with `-Zreg-struct-return unset` in dependency `default_reg_struct_return`
+   = help: set `-Zreg-struct-return unset` in this crate or `-Zreg-struct-return=true` in `default_reg_struct_return`
```

Comment:
this diagnostic is poor. it should be something like
```suggestion
   = help: unset `-Zreg-struct-return` in this crate or set `-Zreg-struct-return=true` in `default_reg_struct_return`
```

Reply:
Fixed.